### PR TITLE
Broken integration test

### PR DESCRIPTION
**Pull request by rhsimplex**
See issue #633 and PR #614.  Basically, a doublespend test that worked before doesn't work anymore.

As far as I can tell, no doublespend occurs (i.e. no duplicate tx in bigchain)

To repeat, though, **a doublespend exception should be raised in this test but isn't**

### CODE REVIEW

**Comment by libscott** [comment]
closes #633

**Comment by libscott** [comment]
This is ready to review now, the test shows that the bug is not reproducible.

**Comment by libscott** [comment]
> Ah... I guess the thing that broke the test earlier was because of colliding tx ids from changes to the model (e.g. removal of timestamp)?

The test wasn't broken, it was trying to reproduce #633. See https://github.com/bigchaindb/bigchaindb/pull/910/commits/9d47fc75acfa5fe6ccbff214bf85c93c42e66d7f#diff-4c7022537015fa4876e90501d3136011R63

I fixed the test to work with the new asset changes so we can close the bug report.

> (Although if that made the test return the double spend as a valid transaction, maybe we should add tests for duplicate transactions too--if we don't already have some?)

That wasn't a double spend, it was the same transaction. `tx_valid` and tx_double_spend` were the same because no timestamp.

**Comment by libscott** [comment]
I don't know what's going on with codecov, the change includes one test but coverage has gone down?

File: tests/integration/test_integration.py [hunk 1]
```diff
@@ -44,3 +44,36 @@ def test_double_create(b, user_pk):
...
+                                           input_valid.id,
+                                           {'2': 2}).sign([user_sk])
+
+    # write the double spend tx
+    b.write_transaction(tx_double_spend)
+    time.sleep(2)
+
+    # still doesn't throw an exception
+    b.get_owned_ids(user_pk)
+    assert b.is_valid_transaction(tx_double_spend) is False
```
Comment (r-marques) [inline tests/integration/test_integration.py]: Can we change this to run `validate_transaction` and check that it is actually failing with a `DoubleSpend`
